src/Main.java

```diff
@@ -1,5 +1,53 @@
+import java.time.LocalDate;
+
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        calculateLeapYear(2021); //Задание 1
+        printInfoAboutVersion(0, 2023);
+        int deliveryDays = calculateDeliveryDistance(95);
+        if (deliveryDays > 0) {
+            System.out.println("Колличество дней доставки " + deliveryDays);
+        }else System.out.println("Доставки нет"); // (Вопрос для консультации)Как сделать так, чтобы изюбежать всей этой требухи(чтобы значени е выводилось непосредственно методом)
```

Comment:
```suggestion
        } else {
            System.out.println("Доставки нет");
        } // (Вопрос для консультации)Как сделать так, чтобы изюбежать всей этой требухи(чтобы значени е выводилось непосредственно методом)
```
фигурные скобки надо всегда писать после if/else

Reply:
ответ на вопрос: вывести в методе. Но по заданию этого делать не нужно